index.bs

```diff
@@ -385,15 +385,16 @@ like
 
 <pre><code class="lang-javascript">
   class ReadableStream {
-    constructor(underlyingSource = {}, { size, highWaterMark } = {})
+    <a href="#rs-constructor">constructor</a>(<a href="#rs-constructor">underlyingSource</a> = {}, { size, highWaterMark
```

Comment:
I realize linking `underlyingSource` to `ReadableStream()` could be seen as questionable. This is just as suggestion. My thinking was that if someone’s looking at the class definition and the constructor, they are most likely a web developer and looking for an interface description. There’s multiple links to `underlyingSource` in the note, so there’s still a path to the definition.